third_party/terraform/utils/transport.go

```diff
@@ -156,7 +156,7 @@ func replaceVars(d TerraformResourceData, config *Config, linkTmpl string) (stri
 		}
 		v, ok := d.GetOk(m)
 		if ok {
-			return v.(string)
+			return fmt.Sprintf("%v", v)
```

Comment:
What's changed that `v` is no longer always a string?

Reply:
Ah! That makes sense. I'd convinced myself this was at import while it was still a string for some reason.